### DISCUSSION TITLE
Killing frame while vertico is active

### DISCUSSION THREAD

**Discussion by haji-ali** [minad/vertico#490] [Q&A] [answered]
I noticed an issue where during a `completing-read` session with `vertico-mode`, in a child frame, if I delete the frame (using `C-x 5 0`), while the completion session is ongoing, the parent frame gets selected but not the minibuffer (the completion session gets transfered to the parent frame). More weirdly, the highlight face get applied to an open buffer in the original frame. Things are even weirder during `consult-buffer` with `vertico` if I delete the frame, since the minibuffer is previewed.

**Comment by minad** [comment, marked as answer]
I think the answer is that you should not kill child frames like this. Simply use C-g. You could also rebind C-x 5 0 if you often hit it it accidentally.
